packages/react-native/ReactCommon/react/renderer/runtimescheduler/SchedulerPriorityUtils.h

```diff
@@ -31,10 +31,9 @@ static inline SchedulerPriority fromRawValue(double value) {
       return SchedulerPriority::LowPriority;
     case 5:
       return SchedulerPriority::IdlePriority;
-    default:
-      react_native_assert(false && "Unsupported SchedulerPriority value");
-      return SchedulerPriority::NormalPriority;
   }
+  LOG(FATAL) << "Unknown SchedulerPriority";
```

Comment:
Theoretically this should be `[[noreturn]]` (RNW Glog stub has dtor be noreturn?)

But not sure MSVC likes that I guess? https://github.com/google/glog/blob/6c5c692c8e423f651c74de9477ff0b5a59008bcc/src/striplog_unittest.cc#L53

Does RNW require the return after this?